internal/containers/provider/docker.go

```diff
@@ -233,6 +236,19 @@ func (p *DockerProvider) Teardown(ctx context.Context) error {
 		}
 	}
 
+	for _, m := range p.req.ManagedVolumes {
+		if mount.TypeVolume != m.Type {
+			continue
+		}
+
+		volume, err := p.cli.VolumeInspect(ctx, m.Source)
+		if err == nil {
+			if err := p.cli.VolumeRemove(ctx, volume.Name, false); err != nil {
+				errs = append(errs, fmt.Errorf("failed to remove volume: %w", err))
+			}
+		}
```

Comment:
this fails silently when `VolumeInspect() fails

maybe just replace this with the standard:

```suggestion
        if err != nil { # handle volume inspect error }
        
		if err := p.cli.VolumeRemove(ctx, volume.Name, false); err != nil {
			errs = append(errs, fmt.Errorf("failed to remove volume: %w", err))
		}
```

Reply:
Ah yes, that's fair. We have a similar thing with the networks, should that be changed as well? 🤔

Reply:
ah! good point. yeah lets fix that up while we're in here then 🙏

Reply:
Welp it doesn't work for networks 😭

Reply:
Fixed this, networks will need more investigation.